core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -1519,6 +1519,12 @@ private CarbonCommonConstants() {
 
   public static final String CARBON_QUERY_PREFETCH_ENABLE_DEFAULT = "true";
 
+  @CarbonProperty
+  public static final String CARBON_QUERY_STAGE_INPUT =
```

Comment:
need support dynamicConfigurable ? else user need to restart the cluster

Reply:
fixed